src/styles/getModalStyles/index.android.js

```diff
@@ -1,7 +0,0 @@
-import getBaseModalStyles from './getBaseModalStyles';
-
-// Only apply top padding on iOS since it's the only platform using SafeAreaView
-export default (type, windowDimensions, popoverAnchorPosition = {}, containerStyle = {}) => ({
-    ...getBaseModalStyles(type, windowDimensions, popoverAnchorPosition, containerStyle),
-    shouldAddTopSafeAreaPadding: false,
```

Comment:
Looking at the attachmentModal code, they have used `statusBarTranslucent={false}` prop.

Reply:
The comment in `getModalStyles/index.android.js` was not right, because we do have safe area on android. Also, this code was causing next bug: 
<img width="535" alt="Screenshot 2022-07-18 at 09 55 56" src="https://user-images.githubusercontent.com/44551710/179467870-1100e420-7aa1-4c7d-89ac-0c21f57c345d.png">

Reverted the deleted file, and added the prop that you have suggested.